src/themes/default/components/core/blocks/Cms/Block.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div
+    :class="['cms-content', { 'container': sync }]"
+    v-if="data"
+    v-html="data.content"
+  />
+</template>
+
+<script>
+export default {
+  name: 'CmsBlock',
+  props: {
+    id: {
+      type: Number,
+      default: null,
+      required: false
+    },
+    identifier: {
+      type: String,
+      default: null,
+      required: false
+    },
+    sync: {
+      type: Boolean,
+      default: false,
+      required: false
+    }
+  },
+  asyncData ({ store, route, context }) {
+    // @TODO to cover SSR need find a way to pass props identifier/id to the asyncData()
```

Comment:
asyncData() won’t be called for child components; so if You place 3 different CmsBlocks on one single page - asyncData won’t be called for these CmsBlocks. Therefore we need this dataLoader pattern implemented - and then we can call store.dispatch(‘dm/registerPromise’, this.loadData()) in created() event handler as we discussed with @yuriboyko on Slack

Reply:
As i see asyncData() is called for child components in our case but the main problem is what we are not able to use component props to fetch correspond block data by identifier. Suggested solution by @pkarw is stll in the progress.  For now I suggest this using SSR for CMS blocks it as a stuff for next pull request

Reply:
As i mentioned. For now CMS blocks recoded to not use SSR as solution initially suggested  does not work and the way with loading all blocks also is not good.  Need find a new solution and it will be a new PR.